### PR TITLE
Add Actions for notifying the other user when a match is dissolved

### PR DESCRIPTION
**Pull request by Jonasdoubleyou**

### CODE REVIEW

File: common/match/dissolve.ts [hunk 1]
```diff
@@ -39,6 +39,12 @@ export async function dissolveMatch(match: Match, dissolveReason: number, dissol
 
     await Notification.actionTaken(student, 'tutor_match_dissolved', { pupil, matchHash, matchDate, uniqueId });
     await Notification.actionTaken(pupil, 'tutee_match_dissolved', { student, matchHash, matchDate, uniqueId });
+
+    if (dissolver.email === student.email) {
+        await Notification.actionTaken(pupil, 'tutee_match_dissolved_other', { student, matchHash, matchDate, uniqueId });
```
Comment (dhenkel92) [inline common/match/dissolve.ts:44]: What's the difference to the `tutee_match_dissolved` notification above?

Comment (Jonasdoubleyou) [reply common/match/dissolve.ts:44]: This one is only triggered for the other person of the match, i.e. when I resolve a match then only my match partner is notified (I don't need to be notified as I triggered the action)

Comment (Jonasdoubleyou) [reply common/match/dissolve.ts:44]: No, that depends on the notifications attached to the action.

Comment (Jonasdoubleyou) [reply common/match/dissolve.ts:44]: We will probably replace tutee_match_dissolved with tutee_match_dissolved_other in the current notification

Comment (dhenkel92) [reply common/match/dissolve.ts:44]: > No, that depends on the notifications attached to the action.

So then it's not a no, but a maybe 😅 



> We will probably replace tutee_match_dissolved with tutee_match_dissolved_other in the current notification

Okay, I see

**Review by dhenkel92 — APPROVED** [review]

**Pull request merged**